File: service-flight-booking-java/src/main/java/com/acme/services/flightBooking/java/Worker.java
package com.acme.services.flightBooking.java;

import io.infinitic.workers.InfiniticWorker;

public class Worker {
    public static void main(String[] args) {
        // infinitic.yml resource is from the dependant "contracts" module
        try (InfiniticWorker worker = InfiniticWorker.fromConfigResource("/infinitic.yml", "/flightBooking.yml")) {
            worker.start();
        }
    }
}

File: service-flight-booking-kotlin/src/main/kotlin/com/acme/services/flightBooking/kotlin/Worker.kt
package com.acme.services.flightBooking.kotlin

import io.infinitic.workers.InfiniticWorker

fun main() {
    // infinitic.yml resource is from the dependant "contracts" module
    InfiniticWorker.fromConfigResource("/infinitic.yml", "/flightBooking.yml").use { worker ->
        worker.start()
    }
}
